routers/api/v1/repo/branch.go

```diff
@@ -117,17 +117,13 @@ func DeleteBranch(ctx *context.APIContext) {
 	//     "$ref": "#/responses/error"
 	//   "404":
 	//     "$ref": "#/responses/notFound"
-
+	//   "423":
+	//     "$ref": "#/responses/repoArchived"
```

Comment:
Please also add a description, here and everywhere else:
```suggestion
	//     "$ref": "#/responses/repoArchived"
	//    "description": "The repo is archived"
```

Reply:
The description is added by swagger automatically